Jenkinsfile

```diff
@@ -66,15 +59,15 @@ def BuildVersionX(publishingRepo, version) {
 
 node {
     def gradleCommand = "./gradlew --info --stacktrace "
-    def publishingRepo = getPublishingRepository()
+    def isPublishNeeded = isPublishNeeded()
```

Comment:
Is there still any functionality that makes a distinction between publishing snapshots and release artifacts? Does Sonatype just detect it correctly?

Reply:
The logic is moved to gradle (That's something in general we want to do)